src/main.rs

```diff
@@ -291,6 +291,17 @@ fn parse_matches() -> clap::ArgMatches {
                 .value_name("NUM")
                 .help("Client Config. Optional: Wait for NUM nodes to converge"),
         )
+        .arg(
+            Arg::with_name("run_client")
+                .long("run-client")
+                .help("Run the client(s)"),
+        )
+        // Heterogeneous Cluster Config
+        .arg(
+            Arg::with_name("no_bootstrap")
+                .long("no-bootstrap")
+                .help("Do not deploy a bootstrap validator. Used when deploying heterogeneous clusters"),
+        )
```

Comment:
Just so I'm sure I understand, the assumption is that using `--no-bootstrap` requires that the bootstrap information is provided in `--cluster-data-path`, correct?

If that's the case, would it simplify things to have this parameter change to `--bootstrap-identity` or `--genesis` and you can pass those in here rather than counting on the cluster data directory? Or are there too many things required in the cluster data directory when using a heterogeneous cluster?

Feel free to push back on any of this, since I'm very likely missing something

Reply:
`no-bootstrap` just ensures you don't build a new genesis and deploy a new bootstrap validator. it does require the faucet account account and the bootstrap account to be valid. so maybe i can replace `--no-bootstrap` with `--bootstrap-identity` and `--faucet-identity`

Reply:
Gotcha, sounds like it might be better to leave this as is rather than add a bunch of new flags -- that can be done later if needed